rust/src/transformations/make_stable_expr/expr_boolean_function/test.rs

```diff
@@ -0,0 +1,236 @@
+use crate::domains::{LazyFrameDomain, OptionDomain, SeriesDomain};
+use crate::metrics::SymmetricDistance;
+
+use super::*;
+
+fn get_f64_i64_data() -> Fallible<(LazyFrameDomain, LazyFrame)> {
+    let lf_domain = LazyFrameDomain::new(vec![
+        SeriesDomain::new("f64", AtomDomain::<f64>::default()),
+        SeriesDomain::new("i64", AtomDomain::<i64>::default()),
+        SeriesDomain::new("f64_null", OptionDomain::new(AtomDomain::<f64>::default())),
+        SeriesDomain::new("i64_null", OptionDomain::new(AtomDomain::<i64>::default())),
+    ])?;
+
+    let lf = df!(
+        "f64" => [0., f64::NAN, f64::NAN, f64::INFINITY],
+        "i64" => [0, 1, 2, 3],
+        "f64_null" => [Some(0.), Some(f64::NAN), None, Some(f64::INFINITY)],
+        "i64_null" => [Some(0), None, None, Some(3)],
+    )?
+    .lazy();
+
+    Ok((lf_domain, lf))
+}
+
+macro_rules! is_nullable {
```

Comment:
In another PR there's a comment about nullable vs. nanable (for lack of a better name). I'm not sure if this is something that would change.